mjtest-files/compile/bench/SubsetSum.mj

```diff
@@ -0,0 +1,193 @@
+class A {
```

Comment:
Could you add a short description what is tested here, similar to the other benchmarks?

Reply:
Thanks!